### DISCUSSION TITLE
IPC Error on channel: connect

### DISCUSSION THREAD

**Discussion by Karakal20** [garrylachman/ElectroCRUD#174] [Q&A]
Please help, when I try to test a connection with ElectronCRUD V3 to SQL Server or PostgreSQL, I get an error called...
"IPC Error on channel: connect"
{"type":1,"message":"{}"}

**Comment by martinconnolly-ffl** [comment]
I have the same error with SQL Server. Also the account creation form does not let me use a blank username, which is allowable for SQL Server.

**Reply by Karakal20** [reply]
Please, if you get the solution to the problem, post it right here, I already asked the creator of ElectroCRUD garrylachman for help directly through the Open Collective web site.

In Postgresql 15.4 what it tells me is that I don't have the correct Driver, but I don't know which driver is mentioned either... With ElectroCRUD v2.8 I connect without problem.

**Comment by Karakal20** [comment]
I just installed SQLite Version sqlite-tools-win32-x86-3430000, it connected on the first try, because it does not require a user, psw, port or anything, just attach the DB.

**Comment by pancaz** [comment]
error test on mysql xampp 7.4 mac os cateline
{"type":1,"message":"{\"code\":\"PROTOCOL_INCORRECT_PACKET_SEQUENCE\",\"fatal\":true}"}

**Comment by didiandalucia** [comment]
I have the same problem with linux mint to an percona mysql server 5.7.44. 
I get same error message, but when I install the old version 2.8 then the connection is perfect possible! 
So thats a big bug in the newest version of ElectroCrud - I am very sorry about
